packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -127,7 +149,12 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
   );
 
   return (
-    <ProtectedPage seo={seo} fallback={<></>} shouldFallback={!user}>
+    <PageComponent
```

Comment:
We also have `ConditionalWrapper` that can handle these kind of things, leaving it up to you what to use though.

Reply:
Hmm yeah, but would not work since `ProtectedPage` requires `seo`, and I would have to change the interface to support it and then again move more things around. Would prefer keeping it here as current.

Reply:
No problem 🙏